packages/phone-number-privacy/combiner/src/config.ts

```diff
@@ -150,6 +159,7 @@ if (DEV_MODE) {
         currentVersion: Number(functionConfig.pnp_keys.current_version),
         versions: functionConfig.pnp_keys.versions,
       },
+      timeoutMs: Number(process.env.TIMEOUT_MS ?? FULL_NODE_TIMEOUT_IN_MS),
```

Comment:
I know I said to pull this from an env variable, but what I meant was actually to pull it from the config. That's my bad for not being clear. Could you please change this to `Number(functionConfig.blockchain.timeout_ms ?? FULL_NODE_TIMEOUT_IN_MS)` 

Note that capital letters aren't allowed in the functionConfig field names